smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/utils/ModelFileUtils.kt

```diff
@@ -0,0 +1,12 @@
+package software.amazon.smithy.swift.codegen.utils
+
+import software.amazon.smithy.swift.codegen.SwiftSettings
+
+class ModelFileUtils {
```

Comment:
This is the utility / helper method used to select a filename for a model source file.